src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -1143,16 +1143,21 @@ internal void RecordAndDeleteModels(List<ModelBase> models)
                         var node = model as NodeModel;
                         Debug.Assert(Nodes.Contains(node));
 
+                        bool silentFlag = node.RaisesModificationEvents;
+                        node.RaisesModificationEvents = false;
```

Comment:
In future if someone introduces a return statement, then we are in trouble. Can we use smart pointers :tada: ??